it/controllers/ValidateIncorporatedEntityDetailsControllerISpec.scala

```diff
@@ -76,7 +78,19 @@ class ValidateIncorporatedEntityDetailsControllerISpec extends ComponentSpecHelp
         val result = post("/validate-details")(suppliedJson)
 
         result.status mustBe BAD_REQUEST
-        result.json mustBe testJson
+        result.json mustBe expectedNoFoundResponseJson
+      }
+
+      "the user asserts that the unincorporated association does not have a Ct Utr, but one is found" in {
+        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+        stubGetCtReference(testCompanyNumber)(status = OK, body = Json.obj("CTUTR" -> testCtutr))
+
+        val suppliedJson = Json.obj("companyNumber" -> testCompanyNumber)
+
+        val result = post(uri = "/validate-details")(suppliedJson)
+
+        result.status mustBe BAD_REQUEST
+        result.json mustBe expectedNoFoundResponseJson
       }
     }
     "return Unauthorised" when {
```

Comment:
Richard I know that was already there but below the line .... 
        stubGetCtReference(testCompanyNumber)(status = OK, body = Json.obj("CTUTR" -> testCtutr))
is not necessary because we stop before due to the fact is not authorised.

Reply:
Done